TIKEBook_Lesson1.ipynb

```diff
@@ -0,0 +1,1247 @@
+{
```

Comment:
<p>On my second read, I wonder if we should explain that 𝑀☉= mass of the sun? (likewise for the "earth" symbol) Although the primary audience is astronomers, we want this to be accessible to data scientists as well.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>added an explanation</p>